slash-commands/slash-commands.js

```diff
@@ -115,14 +149,23 @@ CandyShop.SlashCommands = (function(self, Candy, $) {
 	self.part = function() {
 		Candy.Core.Action.Jabber.Room.Leave(self.currentRoom());
 	};
+	
+	/** Function: leave
+	 * Exits the current chat room
+	 *
+	 */
+	self.leave = function() {
+		Candy.Core.Action.Jabber.Room.Leave(self.currentRoom());
+	};
 
 	/** Function: topic
 	 * Sets the topic (subject) for the current chat room
 	 *
 	 * Parameters:
-	 * 	(String) topic The new topic for the room
+	 *    (String) topic The new topic for the room
 	 */
 	self.topic = function(topic) {
+		// TODO: .isModerator() && allowed checks
```

Comment:
You shouldn’t check if you are moderator, as some rooms allow everyone to change the topic.
Please remove that comment.

Although I’m not sure how errors are displayed currently, this could be investigated.